app/services/legal_framework/add_proceeding_service.rb

```diff
@@ -10,10 +10,7 @@ def call(params)
       @ccms_code = params[:ccms_code]
 
       ActiveRecord::Base.transaction do
-        default_attrs = proceeding_attrs
-        attrs = Setting.enable_loop? ? default_attrs : default_attrs.merge(scope_limitation_attrs)
-        p = Proceeding.create(attrs)
-        add_scope_limitations(p)
+        Proceeding.create(proceeding_attrs)
```

Comment:
although it is was already doing this it was also using the return response, which in the case of a failure would cause further failure i imagine.

According to my understanding and [this article](https://www.bigbinary.com/books/learn-rubyonrails-book/activerecord-transactions-in-depth#exception-handling-and-rolling-back) we should use a bang, yes, or check the response because we are not now using the response `p`

Reply:
Done :+1: